index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet" />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <title>Capstone 1</title>
+  </head>
+  <body>
+    <main class="sub-menu">
+      <img src="/assets/facebook.png" alt="facebook">
+      <img src="/assets/twitter.png" alt="twitter">
+      <span>English</span>
+      <span>My Page</span>
+      <span>Logout</span>
+    </main>
+    <!-- header section -->
+    <header>
+      <nav>
+        <img src="/assets/menu.png" class="menu" alt="menu">
+        <!-- desktop-menu -->
+        <div class="content">
+          <a href="index.html"><img src="/assets/logo.png" alt="logo"></a>
+          <ul class="menu-list">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+          </ul>
+          <a href="" class="camp">GDC Campaign</a>
+        </div>
+        <!-- mobile menu -->
+        <div class="contents" id="contentLink">
+          <img src="/assets/close.png" id="close" alt="close">
+          <a href="index.html" onclick="hideMenu()"><img src="/assets/logo.png" alt="logo"></a>
+          <ul class="menu-list">
+            <li><a href="about.html" onclick="hideMenu()">About</a></li>
+            <li><a href="#" onclick="hideMenu()">Program</a></li>
+            <li><a href="#" onclick="hideMenu()">Join</a></li>
+            <li><a href="#" onclick="hideMenu()">Sponsor</a></li>
+            <li><a href="#" onclick="hideMenu()">News</a></li>
+          </ul>
+          <a href="" class="camp" onclick="hideMenu()">GDC Campaign</a>
+        </div>
+      </nav>
+      <div class="hero-text">
+        <div class="text">
+          <p>"Hello! Sharing world"</p>
+          <p class="title">GAME DEVELOPERS CONFERENCE SHOWCASE 2022</p>
+        </div>
+        <p class="sub-title"> A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in March, in USA.</p>
+        <div class="date-venue">
+          <p class="date">2022-03-21(MON) ~ 25(FRI)</p>
+          <p class="venue">@ Moscone Center San Francisco, CA USA</p>
+        </div>
+      </div>
```

Comment:
Kindly increase the font-size for the mobile view of your heading as it seems really small and not very legible compared to the design given 

design given :
![image](https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/fcac9cf4-2ae8-421a-b075-61a5e269f89a)

Figma design :
![image](https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/6496eb41-5914-482e-95e3-e7af7774c66c)